kani-driver/src/args.rs

```diff
@@ -343,6 +347,16 @@ pub enum ConcretePlaybackMode {
     InPlace,
 }
 
+#[derive(Copy, Clone, Debug, PartialEq, Eq, ValueEnum, strum_macros::Display)]
+#[strum(serialize_all = "kebab-case")]
+pub enum UnstableFeatures {
+    /// Allow replace certain items with stubs (mocks).
```

Comment:
```suggestion
    /// Allows to replace certain items with stubs (mocks).
```
Not sure if it's grammatically correct though.

Reply:
Perhaps
```suggestion
    /// Allow replacing certain items with stubs (mocks).
```